src/modelmanager.cpp

```diff
@@ -47,6 +48,13 @@ namespace ovms {
 
 static bool watcherStarted = false;
 
+ModelManager::ModelManager() {
+    this->customNodeLibraryManager = std::make_unique<CustomNodeLibraryManager>();
+}
+
+ModelManager::~ModelManager() {
```

Comment:
Minor:
You could use
~ModelManager() = default;

Reply:
Done